src/mito2/src/memtable/version.rs

```diff
@@ -38,8 +38,11 @@ impl MemtableVersion {
         }
     }
 
-    /// Returns the mutable memtable.
-    pub(crate) fn mutable(&self) -> &MemtableRef {
-        &self.mutable
+    /// Lists mutable and immutable memtables.
+    pub(crate) fn list_memtables(&self) -> Vec<MemtableRef> {
```

Comment:
Do we still need to maintain a list of immutable memtables?

Reply:
No, I have changed this in my next PR #2262